docs/guides/other-guides/forwarding-and-load-balancing-with-cloud-endpoints.mdx

```diff
@@ -148,46 +61,33 @@ and most common use cases: forwarding traffic to other endpoints
 	}}
 />
 
-## **Step 4** — Create a ngrok Cloud Endpoint
+## **Step 4** — Start an agent session
 
-Now we can create a ngrok Cloud Endpoint that points to our
-newly created internal endpoint. Cloud Endpoints can be created
-via the API or ngrok dashboard.
+At this point, you have a public endpoint with a domain and a traffic policy routing to the URL we defined in our traffic policy above, `https://the-internal-endpoint.internal`. But that URL doesn't exist on the public internet; we need to start an agent session so that requests to our public endpoint will be routed to our internal endpoint, served by the ngrok agent. So, locally, start an agent session with:
 
-<Tabs groupId="guide">
-<TabItem value="api" label="API" default>
-We use the URL from Step 2 as the `url` value in the traffic policy, and the traffic policy from Step 3 as the `traffic-policy` value.
-The binding is set to public so anyone can access our cloud endpoint.
+`ngrok http 80 --url=https://the-internal-endpoint.internal`
 
-```bash
-ngrok api endpoints create \
-    --api-key YOUR_NGROK_API_KEY \
-    --bindings public \
-    --description "sample description" \
-    --metadata sample-metadata \
-    --url https://${NGROK_SUBDOMAIN}.ngrok.app
-    --traffic-policy '{"on_http_request":[{"actions":[{"type":"forward-internal","config":{"url":"https://clep.internal","binding":"internal"}}]}]}'
-```
+Make sure you have some sample application running on port 80.
 
-- Replace NGROK_SUBDOMAIN with the value used in step 2.
+## **Step 5** — See your endpoint in action
```

Comment:
I would make this a separate section called "Load balancing internal endpoints". It's not really a step in the guide, as the user is already done. Also, the previous step is already labeled Step 5